solutions/kubernetes/io-testing.sh

```diff
@@ -22,6 +22,43 @@ set -euo pipefail # exit on failures
 
 source /var/tmp/functions.sh
 
+SOLUTION_FILE="/root/deploy-scripts/k8_cortx_cloud/solution.yaml"
+
+add_separator Setup awscli.
+
+echo -e "\nInstall awscli:-\n"
+pip3 install awscli
+pip3 install awscli-plugin-endpoint
+
+if ! which aws; then
+  add_separator "AWS CLI installation failed"
+  exit 1
+fi
+
+access_key=$(yq e '.solution.common.s3.default_iam_users.auth_admin' $SOLUTION_FILE)
+secret_key=$(yq e '.solution.secrets.content.s3_auth_admin_secret' $SOLUTION_FILE)
+endpoint_url="http://""$(kubectl get svc | grep cortx-io | awk '{ print $3 }')"":8000"
+mkdir -p /root/.aws/
+
+echo -e "\nSetup aws s3 plugin endpoints:-\n"
+aws configure set plugins.endpoint awscli_plugin_endpoint
+check_status "Failed to set awscli s3 plugin endpoint."
+echo -e "\nSetup aws s3 endpoint url:-\n"
+aws configure set s3.endpoint_url $endpoint_url
+check_status "Failed to set awscli s3 plugin endpoint."
```

Comment:
failure message should be `Failed to set awscli s3 endpoint url` to be specific.

Reply:
Done